src/modules/buildShows.js

```diff
@@ -0,0 +1,35 @@
+const buildShows = (showsList, container) => {
+  for (let i = 0; i <= showsList.length - 1; i += 1) {
+    const itemContainer = document.createElement('div');
+    itemContainer.id = `movie${showsList[i].id}`;
+
+    const img = document.createElement('img');
+    img.src = showsList[i].image.medium;
+    img.style.width = '20rem';
+    itemContainer.appendChild(img);
+
+    const titleLike = document.createElement('div');
+    const title = document.createElement('h3');
+    const likeBtn = document.createElement('i');
+    likeBtn.classList.add('bx');
+    likeBtn.classList.add('bx-heart');
+    likeBtn.classList.add('bx-sm');
+
+    title.textContent = `${showsList[i].name}`;
+
+    titleLike.style.margin = '5px';
+    titleLike.style.textAlign = 'center';
+
+    titleLike.appendChild(title);
+    titleLike.appendChild(likeBtn);
+    itemContainer.appendChild(titleLike);
+
+    const commentBtn = document.createElement('div');
+    commentBtn.innerHTML = '<button>Comment</button>';
```

Comment:
why don't we try adding a button identifier such as an attribute, a class, or an id? This would be great in this context for writing more efficient code. It will also be better for linking the popup without having to search for the main container `parent-id`.
It can be either the index of the item in the array or the id of the item in the API data element such as this piece of code you did earlier.
https://github.com/AyoubMs/api-capstone-project/blob/a198d03c1a0647091d19dc67f26f8e8cdac36c63/src/modules/buildShows.js#L4